README.md

```diff
@@ -58,6 +59,20 @@ $ meteor add universe:i18n
 $ meteor npm install --save @types/meteor-universe-i18n
 ```
 
+## Migration to v2 / Breaking changes
+- We removed `locales` and `currency data` from our package. 
+  Delete from your code the following functions: `parseNumber`, `getLanguages`, `getCurrencyCodes`, `getCurrencySymbol`, `getLanguageName`, `getLanguageNativeName` and `isRTL`
+- Remove `_purify` option ([explanation](https://github.com/vazco/meteor-universe-i18n/issues/144#issuecomment-1006522371))
+- Remove `createTranslator` and `createReactiveTranslator`
```

Comment:
It should be phrased differently: "`i18n.locales` got removed", not "We removed `locales`.". Also, we definitely have to add a link to the v2 issue (and PR maybe?), as well as a comment on how to migrate it (e.g., use Intl API or copy-paste the list from v1).